test/integration_tests/configs/suites/quick/offline-er.json

```diff
@@ -5,6 +5,6 @@
     "dataset": "cifar10.json",
     "backend": "local",
     "job_name": "class-incremental-mlp-offline-er",
-    "expected_accuracy": [0.7089999914169312, 0.5120000243186951],
```

Comment:
why is the expected accuracy changing? This should not be the case, no?

Reply:
The behavior of Offline-ER is changed slightly by this PR. In the first update, it will now use the total batch size exclusively for the current task data, since no memory is available yet. Hence, small changes are to be expected and I updated the numbers here.